osf_tests/factories.py

```diff
@@ -341,7 +341,16 @@ class Meta:
     @classmethod
     def _create(cls, *args, **kwargs):
         user = kwargs.pop('creator', None)
-        obj = cls._build(*args, **kwargs)
+        _id = kwargs.pop('_id', None)
+        try:
+            obj = cls._build(*args, **kwargs)
+        except Exception as e:
```

Comment:
Is there a more specific exception that can be caught here? IntegrityError, maybe?